geometryAbstractPractice/src/test/java/comp/fundacionjala/geometryAbstractPractice/RectangleTest.java

```diff
@@ -0,0 +1,32 @@
+package comp.fundacionjala.geometryAbstractPractice;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by HENRRY on 16/06/2016.
+ */
+public class RectangleTest {
+
+    private Rectangle rectangle;
+    private static final double perimeterExpected = 12;
```

Comment:
This variables are only used in the test method
So they should be defined in the test method.